jerry-core/ecma/operations/ecma-proxy-object.h

```diff
@@ -37,10 +37,8 @@ ecma_proxy_create_revocable (ecma_value_t target,
                              ecma_value_t handler);
 
 ecma_value_t
-ecma_proxy_revoke_cb (const ecma_value_t function_obj,
-                      const ecma_value_t this_val,
-                      const ecma_value_t args_p[],
-                      const uint32_t args_count);
+ecma_proxy_revoke_cb (ecma_object_t *function_obj_p,
+                      const ecma_value_t args_p[], const uint32_t args_count);
```

Comment:
This indentation is strange. To create the pattern in the file please move all arguments into single lines.